### DISCUSSION TITLE
How to fix AttributeError: 'Block' object has no attribute 'drop_path'

### DISCUSSION THREAD

**Discussion by Ereshkigal0** [Extraltodeus/multi-subject-render#55] [Q&A]
I have all the settings from the tutorial part of this Extension's stuff, my prompts are pretty basic, I had to remove some things to stop getting a different error (which was TypeError: '>' not supported between instances of 'NoneType' and 'int' ), and I thought it was finally going to work, but then it got to the part where it's done rendering the background & my two subjects separately, and when it goes to put them together... it spits out this error every time:

AttributeError: 'Block' object has no attribute 'drop_path'

Has anyone gotten this error & then fixed it? If so, what do I do? I've already tried replacing the MiDaS folder with the master one, both ways (copying the files directly in to the existing folder, and deleting the existing folder & putting the new one in the same spot, neither fixed the error).

Any help would be greatly appreciated, I was really looking forward to using this extension to render two distinct models together, but haven't been able to get it working yet, and whenever I try to do it with just regular prompts, it blends the two faces together I'm trying to work with so that both of them look kinda the same, instead of one being one person, and the other being the other person.

**Comment by Extraltodeus** [comment]
I close this ticket since it is related to this issue:

https://github.com/Extraltodeus/multi-subject-render/issues/54#issuecomment-1813106133

I am sorry that you can not use it for now. I will take a look as soon as I am able.
